keystore/keystore.go

```diff
@@ -47,8 +47,11 @@ const (
 )
 
 var (
-	ErrStorageLock     = fmt.Errorf("Unable to acquire storage lock")
-	ErrStorageUnlocked = fmt.Errorf("Storage is not locked")
+	ErrStorageLock     = fmt.Errorf("unable to acquire storage lock")
+	ErrStorageUnlocked = fmt.Errorf("storage is not locked")
+	ErrKeyNotInCache   = fmt.Errorf("public key not found in the cache.  Maybe it's not unlocked")
+	ErrKeyNotFound     = fmt.Errorf("public key not found in the key store")
+	ErrInvalidEncData  = fmt.Errorf("Invalid encrypted data")
```

Comment:
If the commit says `don't use capitals in errors` maybe the ErrInvalidEncData should be in lowercase also?

Reply:
thanks for spotting this!